src/SFA.DAS.AssessorService.Application/Interfaces/IScheduleRepository.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using SFA.DAS.AssessorService.Domain.Entities;
 
 namespace SFA.DAS.AssessorService.Application.Interfaces
 {
     public interface IScheduleRepository
     {
+        Task<ScheduleRun> GetScheduleRun(Guid scheduleRunId);
+        Task<IEnumerable<ScheduleRun>> GetAllScheduleRun(int scheduleType);
```

Comment:
Hmmm, and GetNextScheduleToRunNow is missing from the Repo.  Did you pull latest master before starting? :)